dashboard/app/views/pd/workshop_user_admin/facilitator_courses_form.html.haml

```diff
@@ -0,0 +1,47 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/pd', media: 'all'
+
+%h1 Manage Facilitator Courses
+
+#application-container.selectable
```

Comment:
Done.  I'll create a separate PR for removing all of the legacy `.selectable` declarations.